server/middleware/auth.js

```diff
@@ -12,8 +12,8 @@ const auth = (req, res, next) => {
             return;
         } else {
             const decodedToken = jwt(jwtToken);
-            const { userId } = decodedToken;
-            req.userId = userId;
+            const { user_id } = decodedToken;
```

Comment:
I think we agreed during the standup yesterday to use camel case for userId. You might need to change in the `authentication.js` how the token is generated